docs/reference/search/suggesters/completion-suggest.asciidoc

```diff
@@ -202,24 +202,66 @@ The configured weight for a suggestion is returned as `_score`. The
 `text` field uses the `input` of your indexed suggestion. Suggestions
 return the full document `_source` by default. The size of the `_source`
 can impact performance due to disk fetch and network transport overhead.
-For best performance, filter out unnecessary fields from the `_source`
+To save some network overhead, filter out unnecessary fields from the `_source`
 using <<search-request-source-filtering, source filtering>> to minimize
-`_source` size. The following demonstrates an example completion query
-with source filtering:
+`_source` size. Note that the `_suggest` endpoint doesn't support this. The
```

Comment:
> Note that the `_suggest` endpoint doesn't support this

Maybe something like

Note that the `_suggest` endpoint doesn't support source filtering but using `suggest` on the `_search` endpoint does.